sorc/global_cycle.fd/cycle.f90

```diff
@@ -502,6 +525,51 @@ SUBROUTINE SFCDRV(LUGB, IDIM,JDIM,LENSFC,LSOIL,DELTSFC,  &
  ENDIF
 
 !--------------------------------------------------------------------------------
+! READ IN AND APPLY LAND INCEREMENTS FROM THE GSI
+!--------------------------------------------------------------------------------
+
+ IF (DO_LNDINC) THEN 
+
+!--------------------------------------------------------------------------------
+! RE-CALCULATE SOILSNOW MASK AFTER SNOW UPDATE
+!--------------------------------------------------------------------------------
+
+    IF (DO_SFCCYCLE)  THEN ! should really make this a snow DA flag, but this will do
+        CALL CALCULATE_SOILSNOWMASK(SLCFCS(:,1),SNOFCS, LENSFC, SOILSNOW_MASK ) 
+    ELSE 
+        SOILSNOW_MASK = SOILSNOW_FG_MASK
+    ENDIF 
+
+!--------------------------------------------------------------------------------
+! read increments in 
+!--------------------------------------------------------------------------------
+
+    INQUIRE(FILE=trim(LND_FILE), EXIST=file_exists)
+    IF (.not. file_exists) then 
+        print *, 'ERROR: land increment update requested, but file does not exist: ', & 
```

Comment:
@ClaraDraper-NOAA NCEP Central Operations (NCO) requires all messages associated with job termination to begin with "FATAL ERROR".  I got in trouble for this recently (see #368).

Reply:
Just fixed that, and pushed it.